package.json

```diff
@@ -81,7 +81,7 @@
     "@types/jasminewd2": "~2.0.3",
     "@types/jquery": "^3.3.15",
     "@types/jszip": "^3.1.5",
-    "@types/node": "^6.0.60",
+    "@types/node": "^10.14.0",
```

Comment:
Should this be in sync with the node version in .nvmrc, v10.13.0?

I don't know what this is used for, so comment may not apply.

Reply:
The PR comment.  There  just isn't one